PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1268,6 +1271,7 @@ Continue?"/>
                     <Item id="3126" name="Save As..."/>
                     <Item id="3127" name="Save a Copy As..."/>
                     <Item id="3121" name="Add New Project"/>
+                    <Item id="3128" name="Find in Files.../>
```

Comment:
From https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/37827195/job/evdo8fivme4cv4gu/messages

> PowerEditor/installer/nativeLang\english.xml: PowerEditor/installer/nativeLang/english.xml:1275:17:FATAL:PARSER:ERR_LT_IN_ATTRIBUTE: Unescaped '<' not allowed in a

closing " is missing in line 1274

Reply:
@chcg, Thank you. Some errors cannot be detected by simply compiling it. I should perform the xml checks too ...